SU-HSDO/suhumsci
# feat(shs-5072): updates to private component browser images

# READY FOR REVIEW

## Summary
Adds paragraph browser images (and icons) to the Private Collection and Private Text Area components

## Urgency
'medium'

## Steps to Test
1. Visit a page like https://hs-traditional.suhumsci.loc/admin/structure/paragraphs_type and see the Private Collection and Private Text Area thumbnails
2. Edit a Private page like https://hs-traditional.suhumsci.loc/private-pages/private-page-components
3. Click Add Paragraph at the bottom of the page
4. Note the Private Text Area image and the Private Collection image on the Paragraph Browser modal that pops up. 
 
## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
